Tests/Opc.Ua.Client.Tests/ClientTest.cs

```diff
@@ -1705,7 +1705,9 @@ public async Task OpenSessionECCUserCertIdentityToken(
                 }
                 if (eccurveHashPair.Curve.Oid.FriendlyName.Contains(extractedFriendlyNamae))
                 {
+
                     X509Certificate2 cert = CertificateBuilder.Create("CN=Client Test ECC Subject, O=OPC Foundation")
+                    .SetHashAlgorithm(eccurveHashPair.HashAlgorithmName)
```

Comment:
this should not be necessary now, lets keep the tests as similar to production code as possible.

Or am i missing something here?